composer/trainer/trainer.py

```diff
@@ -1883,7 +1890,8 @@ def _train_loop(self) -> None:
 
         use_grad_scaling = self._use_grad_scaling(self.state.precision, self.state.scaler)
 
-        self._spin_dataloaders()
+        if self.spin_dataloaders:
+            self._spin_dataloaders()
```

Comment:
suggest renaming this to `_forward_dataloaders_to_current_epoch`, `_set_dataloader_rng_to_current_epoch`, or something similar.